app/controllers/emails_controller.rb

```diff
@@ -54,15 +54,22 @@ def save_template
       flash.now[:alert] = t('.saving_failed')
     end
     @event = Event.find(params[:event_id])
-    @templates = EmailTemplate.with_status(get_status)
+    @templates = EmailTemplate.with_status(get_email_template_status)
 
     render :email
   end
 
-  def get_status
+  def get_email_template_status
     params[:status] ? params[:status].to_sym : :default
   end
 
+  def get_corresponding_application_letter_status
```

Comment:
You are right, we rarely test private methods.  But I think test should beside guaraneing that the public interface works, minimize the effort to find the bug if something has broken (I don't like debugging). If the private method is broken a test would point directly to the broken location. I think for consistency reasons it is OK in this PR, but maybe we could talk about it later? @Baschdl?